README.md

```diff
@@ -1,15 +1,27 @@
 # bitmovin-player-tizen-demo
-This repository contains a sample app that used the Bitmovin HTML5 Player in a Tizen App. 
+
+This repository contains a sample app that used the Bitmovin HTML5 Player in a Tizen App. Project structure:
```

Comment:
```suggestion
This repository contains a sample app that uses the Bitmovin HTML5 Player in a Tizen App. Project structure:
```
also not entirely sure if 
`This repository contains a sample app which uses the Bitmovin HTML5 Player in a Tizen App. Project structure:` ("which" instead of "that") would fit better

Reply:
Did a quick search; `that` seems to fit better because it's a defining clause in that case :thinking:

Reply:
Done :heavy_check_mark: [`f329268` (#12)](https://github.com/bitmovin/bitmovin-player-tizen-demo/pull/12/commits/f3292687af78627a0b69c275140d48b1f732a299)